packages/components/src/components/expandable/expandable.ts

```diff
@@ -0,0 +1,221 @@
+  async show() {
+    if (this.open) {
+      return undefined;
+    }
+
+    this.onToggleClick();
+    return waitForEvent(this, 'sd-after-show');
+  }
+
+  /** Closes the expandable */
+  async hide() {
+    if (!this.open) {
+      return undefined;
+    }
+
+    this.onToggleClick();
+    return waitForEvent(this, 'sd-after-hide');
+  }
+
+  render() {
+    return html`
+      <details part="details">
+        <summary part="summary" aria-hidden="true" class="cursor-pointer overflow-hidden list-none">
+          <slot name="clone"></slot>
+        </summary>
+        <div part="content" class="content content-preview overflow-hidden relative">
+          <slot></slot>
+        </div>
+      </details>
+      <button
+        part="toggle"
+        class=${cx(
+          'sd-interactive sd-interactive--reset !h-full !justify-center !w-full !text-base !flex !items-center !underline !underline-offset-2 !toggle',
+          this.inverted && 'sd-interactive--inverted',
+          !this.inverted ? '!focus-visible:focus-outline' : '!focus-visible:focus-outline-inverted'
+        )}
+        @click=${this.onToggleClick}
+      >
+        ${this.open
+          ? html`
+              <slot name="toggle-open">
+                <sd-icon class="mr-2 text-xl" library="system" name="chevron-up"></sd-icon>
+                ${this.localize.term('showLess')}
+              </slot>
+            `
+          : html`
+              <slot name="toggle-closed">
+                <sd-icon class="mr-2 text-xl" library="system" name="chevron-down"></sd-icon>
+                ${this.localize.term('showMore')}
+              </slot>
+            `}
+      </button>
+    `;
+  }
+
+  firstUpdated() {
+    this.cloneContentToLightDOM();
+
+    // Set the initial state of the details element
+    this.details.setAttribute('open', this.open.toString());
+  }
+
+  cloneContentToLightDOM() {
+    const slot = document.createElement('div');
+    slot.setAttribute('slot', 'clone');
+
+    const nodes = Array.from(this.childNodes);
+    nodes.forEach(node => {
+      const clone = node.cloneNode(true);
+      slot.appendChild(clone);
+    });
+
+    this.appendChild(slot);
+  }
+
+  static styles = [
+    SolidElement.styles,
+    unsafeCSS(InteractiveStyles),
+    componentStyles,
+    css`
+      :host {
+        --gradient-color-start: rgba(255, 255, 255, 0);
+        --gradient-color-end: rgba(255, 255, 255, 1);
+        --component-expandable-max-block-size: 90px;
+        --gradient-height: 24px;
+
+        --gradient: var(--gradient-color-start) 0%, var(--gradient-color-end) 80%, var(--gradient-color-end) 100%;
+
+        @apply inline-block relative w-full;
+      }
+
+      .toggle::-moz-focus-inner {
+        @apply border-none p-0;
+      }
+
+      details > summary::-webkit-details-marker,
+      details[open] summary {
+        @apply hidden;
+      }
+
+      summary {
+        max-block-size: var(--component-expandable-max-block-size, 90px);
```

Comment:
Since the variable is defined in the host, I don't think you need to add the fallback 90px here.